test/carboncopy_proper_tests_SUITE.erl

```diff
@@ -19,50 +19,73 @@ all() ->
     [{group, mod_message_carbons_proper_tests}].
 
 all_tests() ->
-    [chat_type_test, private_message_test, no_copy_type_test,
-          received_type_test, sent_forwarded_type_test, sent_message_test,
-          simple_chat_message_test, simple_badarg_test].
+    [
+     private_message_test,
+     no_copy_type_test,
+     empty_message_test,
+     received_type_test,
+     sent_message_test,
+     simple_badarg_test,
+     simple_chat_message_test,
+     has_chat_state_notifications,
+     has_delivery_receipts,
+     is_muc_invitation,
+     is_direct_invitation
```

Comment:
I know that adding `_test` seems a bit awkward, but maybe either do it always or never in a single test suite? Just to stay consistent.